auth/frontend-react-typescript-next/pages/index.tsx

```diff
@@ -42,8 +54,15 @@ const Home: NextPage = () => {
       </Head>
       <main className={styles.main}>
         <div className={styles.container}>
-          <button type="button" onClick={handleSignInClick}>Sign in with Footprint</button>
-          <button type="button" onClick={handleUpdateAuthMethodsClick}>Update Authentication methods with Footprint</button>
+          <button type="button" onClick={handleSignInClick}>
+            Sign in with Footprint
+          </button>
+          <button type="button" onClick={handleUpdateAuthMethodsClickLegacy}>
+            Update Authentication methods with Footprint (legacy)
```

Comment:
Do we really need to share the legacy example here? I think this will just be confusing to the tenants